modules/lvgl/lvgl.c

```diff
@@ -152,7 +152,11 @@ static int lvgl_allocate_rendering_buffers(lv_disp_drv_t *disp_driver)
 		buf_size = 4 * buf_nbr_pixels;
 		break;
 	case PIXEL_FORMAT_RGB_888:
+#if defined(CONFIG_RENESAS_RA_DISPLAY)
```

Comment:
Don't add driver specific Kconfigs here like this. Why does the display hardware need an additional byte per pixel? Is it using RGB_888 as the display format, or ARGB_8888?

Reply:
The RGB888 input buffer for the Renesas GLCD controller driver requires 32 bits as 1 pixel as shown in the image below. So, each RGB888 pixel buffer should have 1 byte padding at MSB.
![image](https://github.com/user-attachments/assets/010b3900-562d-4e70-a555-00ee7d0488ec)
Because handling the memory padding by software inside the driver code can take a huge CPU load, I think it is better to give it to application prepare/input directly 32bitRGB888 buffer for display API. If it is not acceptable, could you suggest a more suitable way?

Reply:
Could the display format be reported as ARGB8888? It seems like the GLCD would just ignore the upper 8 bits, so and ARGB8888 buffer would be equivalent to an RGB8888 buffer with no alpha channel. It seems like LVGL already simply sets the alpha channel to 0xFF in some cases: https://github.com/zephyrproject-rtos/lvgl/blob/zephyr/src/draw/lv_img_buf.c#L51

Given this, why not just report ARGB8888?

Reply:
Done. I moved the processing into the shim driver code. I have confirmed that the LVGL render buffer fills channel A with 0xFF